qermit/coherent_pauli_checks/box_clifford_subcircuits.py

```diff
@@ -0,0 +1,316 @@
+import networkx as nx  # type: ignore
+from pytket._tket.unit_id import Qubit
+from pytket.circuit import CircBox, Circuit, Command, OpType
+from pytket.passes import BasePass, CustomPass
+
+from .monochromatic_convex_subdag import get_monochromatic_convex_subdag
+
+
+def _command_is_clifford(command: Command) -> bool:
+    """Check if the given command is Clifford.
+
+    :param command: Command to check.
+    :return: Boolean value indicating if given command is Clifford.
+    """
+
+    # This is only a limited set of gates.
+    # TODO: This should be expanded.
```

Comment:
You can use https://docs.quantinuum.com/tket/api-docs/circuit.html#pytket.circuit.Op.is_clifford .

Reply:
I think I will leave it as it is for now. It's not only important that they are Clifford, but also that they are the particular subset of Clifford which my simulator supports. This group here belongs to that set.